src/navigations/HomeBottomTabs.js

```diff
@@ -47,14 +49,14 @@ function HomeBottomTabs() {
   const renderTabBarIcon = (route, focused, color) => {
     if (route.name === 'SignedChannelList') {
       return (
-        <View style={styles.center}>
+        <View style={styles.iconContainer}>
           <SignedChat fill={color} stroke={color} />
         </View>
       );
     }
     if (route.name === 'AnonymousChannelList') {
       return (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the icon container has been changed from `styles.center` to `styles.iconContainer`. This change suggests that the styling of the icon container has been updated, possibly to accommodate new design requirements.
```diff
- <View style={styles.center}>
+ <View style={styles.iconContainer}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->